index.bs

```diff
@@ -2680,6 +2680,22 @@ which meant converting the names to camel-cased version.
 One such example is the <a href="https://w3c.github.io/image-resource/">image resource</a>.
 For this reason, if a key can clearly be expressed as a single word, that is recommended.
 
+<h3 id="serialization">Serialization</h3>
```

Comment:
Title should be an imperative summary of guidance, [per newer meta - design principle](https://github.com/w3ctag/design-principles/pull/371)

Reply:
```suggestion
<h3 id="serialization">Consider Consumers when Serializing</h3>
```